### PR TITLE
feat: add more tags

### PR DESCRIPTION
**Pull request by LunarWatcher**
# Pull Request

## Description:

 
Adds three new tags; win, contribute, and pr-count, the second of which should see some solid regular use.

### CODE REVIEW

File: src/data/tags.ts [hunk 1]
```diff
@@ -117,4 +131,11 @@ export const tags: Tag[] = [
     title: "UWU?",
     aliases: [],
   },
+  {
+    name: "win",
+    content:
+      "Hacktoberfest doesn't have any winners or losers. Positive contributions to the open-source community is a win-win for everyone",
```
Comment (naomi-lgbt) [inline src/data/tags.ts:137]:
```suggestion
      "Hacktoberfest doesn't have any winners or losers. Positive contributions to the open-source community is a win for everyone",
```
Nitpick

File: src/data/tags.ts [hunk 2]
```diff
@@ -70,6 +77,13 @@ export const tags: Tag[] = [
     title: "How does a PR/MR count for Hacktoberfest?",
     aliases: ["pr", "mrs", "mr"],
   },
+  {
+    name: "pr-count",
+    content:
+      "You need four PRs/MRs accepted between October 1 and October 31 to complete Hacktoberfest",
+    title: "How many PRs/MRs do I need?",
+    aliases: ["mr-count"],
```
Comment (naomi-lgbt) [inline src/data/tags.ts:85]:
```suggestion
    aliases: ["mr-count", "count"],
```
Might be good to alias `count` too.

File: src/data/tags.ts [hunk 3]
```diff
@@ -1,6 +1,13 @@
 import { Tag } from "../interfaces/Tag";
 
 export const tags: Tag[] = [
+  {
+    name: "contribute",
+    content:
+      "You can contribute to any repo with the hacktoberfest label, or if your PR gets the hacktoberfest-accepted label. You can find these by searching on either [GitHub](https://github.com/topics/hacktoberfest) or [GitLab](https://gitlab.com/explore/projects/topics/hacktoberfest).\n\nYou can contribute both code and low-code. See also [GitHub's guide](https://opensource.guide/how-to-contribute/) for contributing if you're not sure how. The [Hacktoberfest website](https://hacktoberfest.com/participation/#contributors) also contains some resources to help you get started.",
```
Comment (naomi-lgbt) [inline src/data/tags.ts:7]:
```suggestion
      "You can contribute to any repo with the hacktoberfest label, or repos using the `hacktoberfest-label` on individual PRs/MRs. You can find these by searching on either [GitHub](https://github.com/topics/hacktoberfest) or [GitLab](https://gitlab.com/explore/projects/topics/hacktoberfest).\n\nYou can contribute both code and low-code. See also [GitHub's guide](https://opensource.guide/how-to-contribute/) for contributing if you're not sure how. The [Hacktoberfest website](https://hacktoberfest.com/participation/#contributors) also contains some resources to help you get started.",
```
Tweak the wording here a bit.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Looking good! A couple of nitpicks.

File: src/data/tags.ts (hunk 3 above)
Comment (LunarWatcher) [reply src/data/tags.ts:7]: I assume you typo'd and meant "[...] or repos using the `hacktoberfest-accepted` label on individual [...]"?

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Comment by naomi-lgbt** [comment]
## Thank You

Thank you for your continued contribution to our project. We have reviewed your pull request and are happy to accept these changes.

Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Pull request merged**